modules/administration-guide/nav.adoc

```diff
@@ -70,3 +70,4 @@
 ** xref:configuring-authorization.adoc[]
 ** xref:configuring-openshift-oauth.adoc[]
 ** xref:removing-user-data.adoc[]
+** xref:configuring-minikube-github-authentication.adoc[]
```

Comment:
It rather belongs to *Administration Guide > Managing identities and authorizations > Configuring authorization > Enabling authentication with social accounts and brokering*.